CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   ([PR](https://github.com/hashicorp/boundary/pull/1736))
 * controllers/workers: Add client IP to inbound request information which is included in
   Boundary events ([PR](https://github.com/hashicorp/boundary/pull/1678))
+* plugins/aws: AWS host sets will now return DNS hosts in addition to IP
```

Comment:
I think the terminology here could be confusing for someone not super familiar with boundary (boundary doesn't have the concept of a "dns host", host sets don't return dns values, etc...).  How about something like "AWS plugin based hosts now include dns names in addition to the ip addresses they already provide."

Reply:
Applied suggestion!